app/src/main/java/com/example/parkingsystemjava/fragment/DateTimeReservationDialogFragment.java

```diff
@@ -0,0 +1,23 @@
+package com.example.parkingsystemjava.fragment;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.fragment.app.DialogFragment;
+import com.example.parkingsystemjava.databinding.DialogFragmentDateTimeReservationBinding;
+
+public class DateTimeReservationDialogFragment extends DialogFragment {
+
+    public static DateTimeReservationDialogFragment newInstance() {
+        DateTimeReservationDialogFragment instanceDateTimeReservationDialogFragment = new DateTimeReservationDialogFragment();
```

Comment:
same here you are creating a variable and you only use it in one place you can do this
`return new DateTimeReservationDialogFragment();`

Reply:
Done! I thought I needed to create a variable first, thanks!